pkg/configs/db/db.go

```diff
@@ -70,14 +86,11 @@ func New(cfg Config) (DB, error) {
 	}
 
 	if len(cfg.PasswordFile) != 0 {
-		if u.User == nil {
-			return nil, fmt.Errorf("--database.password-file requires username in --database.uri")
-		}
-		passwordBytes, err := os.ReadFile(cfg.PasswordFile)
+		updatedUrl, err := SetUserPassword(u, cfg.PasswordFile)
```

Comment:
```suggestion
		updatedURL, err := SetUserPassword(u, cfg.PasswordFile)
```
So it passes linting